src/open_inwoner/accounts/views/contactmoments.py

```diff
@@ -211,3 +212,31 @@ def get_context_data(self, **kwargs):
             },
         ]
         return ctx
+
+
+class KlantContactMomentRedirectView(KlantContactMomentAccessMixin, View):
+    """
+    Redirect to `KlantContactMomentDetailView` on the basis of contactmoment uuid
+    """
```

Comment:
Is this view really needed? I'd expect any place we have a contactmoment we'd also have the klantcontactmoment.

It feels weird we need to have a view just for this lookup, especially if we're going to to the same requests at the place it redirects to.

Reply:
Case detail view is getting crowded, so it was suggested to move this into a separate view.

Reply:
Do you have more context then "it was suggested"?

Anyway, the question was why we need a redirect view to do a lookup for information we have both before and after the redirect? I'd expect you could make URLs with the klantcontactmoment uuid.

Reply:
The issue is that if we do this in the case detail view, we would have to do a bunch of fetches to get the associated `KlantContactMoment`en for each `ObjectContactMoment.contactmoment`, because we need the `KlantContactMoment.uuid` to build the URL for the Mijn vraag detail view. Because the case detail view is already quite massive, we figured it would be a better idea to have an intermediate view to resolve this

Reply:
Ah ok. I assumed we only show ContactMomenten for a Zaak for this user's Klant so we'd have the KlantContactMoment, but we show all ContactMomenten for a Zaak so then this makes sense.